src/TestFramework/Config/TestFrameworkConfigLocator.php

```diff
@@ -27,16 +27,17 @@ public function locate(string $testFrameworkName, string $customDir = null): str
 
         foreach (['xml', 'yml'] as $extension) {
             $conf = sprintf('%s/%s.%s', $dir, $testFrameworkName, $extension);
+            $altConf = sprintf('%s/%s.dist.%s', $dir, $testFrameworkName, $extension);
```

Comment:
Could we please do smth like:
```
$conf = sprintf('%s/%s.%s', $dir, $testFrameworkName, $extension);

if (file_exists($conf)) {
    return realpath($conf);
}

if (file_exists($conf . '.dist')) {
    return realpath($conf . '.dist');
}

$altConf = sprintf('%s/%s.dist.%s', $dir, $testFrameworkName, $extension);
if (file_exists($altConf)) {
    return realpath($altConf);
}
```
instead of if-elseif-elseif ?

Reply:
no iffs and just loud and clear
```php
[
    'xml', 'yml',
    'xml.dist', 'yml.dist',
    'dist.xml', 'dist.yml',
]
```